src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements FigureArea {
+
+    public Circle(int sideA) {
+        super(sideA);
+    }
```

Comment:
we should have a field `int side` in the Circle class and do the following in the constructor:
```suggestion
    public Circle(int side, String color) {
        super(color);
        this.side = side;
    }
```